src/python_testing/TC_MOD_1_2.py

```
@@ -158,9 +158,11 @@ async def test_TC_MOD_1_2(self):
         if await self.attribute_guard(endpoint=self.endpoint, attribute=self.cluster.Attributes.StartUpMode):
             startup_mode = await self.read_single_attribute_check_success(endpoint=self.endpoint, cluster=self.cluster, attribute=self.cluster.Attributes.StartUpMode)
             self._log_attribute("StartupMode", startup_mode)
-            asserts.assert_true(isinstance(startup_mode, int), "Startupmode is not int")
-            asserts.assert_in(startup_mode, supported_modes_values,
-                              f"Startupmode {startup_mode} is not in {supported_modes_values}")
+            asserts.assert_true((matter_asserts.is_valid_uint_value(startup_mode, 8) or startup_mode is NullValue, "Value for StartupMode is not uint8 or Null"),
```

Flake8 (7.3.0)

[error] undefined name 'matter_asserts'

(F821)